sigle/src/ui/Button.ts

```diff
@@ -20,9 +20,13 @@ export const Button = styled('button', {
         br: '$1',
       },
       lg: {
-        px: '$4',
-        py: '$3',
+        px: '$3',
```

Comment:
@gregogun every screen will be affected by this change correct? Did you check with Quentin that other screens were okay?

Reply:
yep confirmed this with Quentin :)